dftimewolf/cli/curses_display_manager.py

```diff
@@ -210,23 +213,36 @@ def EnqueueMessage(self,
     if self._messages_longest_source_len < len(source):
       self._messages_longest_source_len = len(source)
 
-    _, x = self._stdscr.getmaxyx()
-
-    lines = []
-
-    # textwrap.wrap behaves strangely if there are newlines in the content
     for line in content.split('\n'):
-      lines += textwrap.wrap(line,
-                             width=x - self._messages_longest_source_len - 8,
-                             subsequent_indent='  ',
-                             replace_whitespace=False)
-
-    for line in lines:
       if line:
         self._messages.append(Message(source, line, is_error))
 
     self.Draw()
 
+  def PrepareMessagesForDisplay(self, space: int) -> List[str]:
```

Comment:
Nit: can you change "space" to something more explicit?
maybe something like "tty_max_lines" ?

Reply:
Done.